bundles/org.openhab.core.model.script/src/org/eclipse/smarthome/model/script/actions/HTTP.java

```diff
@@ -60,6 +62,25 @@ static public String sendHttpGetRequest(String url, int timeout) {
         return response;
     }
 
+    /**
+     * Send out a GET-HTTP request. Errors will be logged, returned values just ignored.
+     * 
+     * @param url the URL to be used for the GET request.
+     * @param headers the HTTP headers to be sent in the request.
+     * @param timeout timeout in ms
+     * @return the response body or <code>NULL</code> when the request went wrong
+     */
+    public static String sendHttpGetRequest(String url, Map<String, String> headers, int timeout) {
+        try {
+            Properties h = new Properties();

```

Comment:
Got any ideas? I can't use "headers". How about propHeaders, or headerProperties? Alternatively do you know if this can be "one-lined" without needing a variable?

Reply:
changed to headerProperties